exotica/include/exotica/KinematicTree.h

```diff
@@ -99,10 +99,10 @@ class KinematicsRequest
 
 struct KinematicFrame
 {
-    std::shared_ptr<KinematicElement> FrameA;
-    KDL::Frame FrameAOffset;
-    std::shared_ptr<KinematicElement> FrameB;
-    KDL::Frame FrameBOffset;
+    std::shared_ptr<KinematicElement> FrameA = nullptr;
+    KDL::Frame FrameAOffset = KDL::Frame();
+    std::shared_ptr<KinematicElement> FrameB = nullptr;
+    KDL::Frame FrameBOffset = KDL::Frame();
     KDL::Frame TempAB;
```

Comment:
I believe all these initializations are redundant. Both ```shared_ptr``` and ```KDL::Frame``` will be initialized to those values by default.